plugins/services/src/js/components/forms/VolumesFormSection.js

```diff
@@ -165,13 +165,19 @@ class VolumesFormSection extends Component {
    * @return {Array} elements
    */
   getExternalVolumesLines(data, offset) {
-    return data.map((volumes, key) => {
+    return data.map((volume, key) => {
       const nameError = errorsLens
         .at(key + offset, {})
         .attr('external', {})
         .get(this.props.errors)
         .name;
 
+      const sizeError = errorsLens
+      .at(key, {})
```

Comment:
our new `indent` eslint rule still doesn't catch that, we should consider `newline-per-chained-call` but would you please add some spaces?